From: bitcoffee<854182924@qq.com>
Reason: pkgconfig install path write dead in the file, modify it
diff -urN grpc/cmake/pkg-config-template.pc.in grpc_new/cmake/pkg-config-template.pc.in
--- grpc/cmake/pkg-config-template.pc.in        2020-08-21 14:34:20.512037605 +0800
+++ grpc_new/cmake/pkg-config-template.pc.in    2020-08-27 20:47:46.372035834 +0800
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 includedir=${prefix}/include
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}/lib64

 Name: @PC_NAME@
 Description: @PC_DESCRIPTION@
diff -urN grpc/CMakeLists.txt grpc_new/CMakeLists.txt
--- grpc/CMakeLists.txt	2020-08-24 09:14:14.361862041 +0800
+++ grpc_new/CMakeLists.txt	2020-08-27 14:50:00.371507303 +0800
@@ -42,6 +42,7 @@
 set(gRPC_INSTALL_INCLUDEDIR "include" CACHE STRING "Installation directory for headers")
 set(gRPC_INSTALL_CMAKEDIR "lib/cmake/${PACKAGE_NAME}" CACHE STRING "Installation directory for cmake config files")
 set(gRPC_INSTALL_SHAREDIR "share/grpc" CACHE STRING "Installation directory for root certificates")
+set(gRPC_INSTALL_PKGCONFIGDIR "lib/pkgconfig" CACHE STRING "Installation directory for pkgconfig")
 
 # Options
 option(gRPC_BUILD_TESTS "Build tests" OFF)
@@ -15666,7 +15667,7 @@
     "${output_filepath}"
     @ONLY)
   install(FILES "${output_filepath}"
-    DESTINATION "lib/pkgconfig/")
+    DESTINATION ${gRPC_INSTALL_PKGCONFIGDIR})
 endfunction()
 
 # gpr .pc file
